### ISSUE TITLE
Route error on Deploying Openshift Easily 

### ISSUE THREAD

**Issue by gbaufake** [hawkular/hawkular-openshift-agent#170]
Following this http://www.hawkular.org/blog/2017/04/05/deploy-hosa-easily.html

I got this error 
````
The Route "hawkular-openshift-agent" is invalid: spec.host: Invalid value: "hawkular-openshift-agent-default..xip.io": host must conform to DNS 952 subdomain conventions
make: *** [openshift-deploy] Error 1
````

My versions are
````
oc v3.5.5.10
kubernetes v1.5.2+43a9be4
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server ip:8443
openshift v3.5.5.10
kubernetes v1.5.2+43a9be4
````

**Comment by jmazzitelli** [comment]
I think the domain names changed in the 3.5. It is no longer .xip, but .nip.

Look in the Makefile that you pull down with wget:

```
HAWKULAR_OPENSHIFT_AGENT_HOSTNAME ?= "hawkular-openshift-agent-${HAWKULAR_OPENSHIFT_AGENT_NAMESPACE}.$(shell oc version | grep 'Server ' | awk '{print $$2;}' | egrep -o '([0-9]{1,3}[.]){3}[0-9]{1,3}').xip.io"
```

Change that .xip to .nip and see what happens.

**Comment by jmazzitelli** [comment]
See: https://github.com/openshift/origin/issues/13020 "Use nip.io instead of xip.io to provide wildcard DNS for oc cluster up, etc"

**Comment by jmazzitelli** [comment]
The nip vs xip was a red-herring. The OS server had a hostname not an IP and the Makefile failed to parse it. I have a PR that hopefully addresses this.

**Comment by jmazzitelli** [comment]
see PR #171

**Comment by jmazzitelli** [comment]
this should be fixed with PR #171
